quickstarts/rest/Search_Grounding.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>Same here, re: pointing out that there are requirements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done.</p>